OKR.md

```diff
@@ -2,6 +2,34 @@
 
 We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
+## 2018 Q4
+
+# A viable alternative to Go IPFS
+
+* P0 - @vasco-santos - DHT is enabled by default and interoperates with Go IPFS DHT
+* P1 - @vasco-santos - IPNS is functional over pubsub and the DHT
+* P0 - @alanshaw - Base32 CIDv1 migration is complete
+* P0 - @alanshaw - A JS IPFS daemon is part of the IPFS gateways
```

Comment:
what does "is part of" mean here?

Reply:
I mean that it is one (or more!) of the nodes that belong to the IPFS gateway.

Obvisously this will require close communication with the infra team and I anticipate working on requirements the infra team may have for an IPFS gateway node (setup?, reporting?, throughput?, stability?). I'm thinking we'll likely want to "soft" launch and gradually ramp up traffic being directed to the node as performance and stability are proven.